docs/raspberry-pi4.md

```diff
@@ -1,125 +1,128 @@
-
-### OpenSSH
-
-Ubuntu Server will deploy and enable [OpenSSH](https://www.openssh.com/) by default, but make sure that for whichever user you're going to deploy the cluster with on the build system their [SSH Key is copied to each node's root user](https://www.cyberciti.biz/faq/use-ssh-copy-id-with-an-openssh-server-listing-on-a-different-port/), or you may have to do additional manual configurations to run the example.
-
-Effectively before you start, you should have it configured so that the current user can run:
+Before you start, the configuration should be such that the current user can run:
 
 ```shell
 $ ssh root@${HOST}
 ```
 
-Where `${HOST}` is any node and the login will succeed with no further prompts.
+Where `${HOST}` is any node and the login can succeed with no further prompts.
 
-## Setup Nodes
+### 3. Set up Nodes
 
-Each node (whether control plane or not) will need some additional setup to prepare for k0s deployment.
+Every node (whether control plane or not) requires additional configuration in
+preparation for k0s deployment.
 
-### CGroup Configuration
+#### CGroup Configuration
 
-Ensure that the following packages are installed on each node:
+1. Ensure that the following packages are installed on each node:
 
-```shell
-$ apt-get install cgroup-lite cgroup-tools cgroupfs-mount
-```
+    ```shell
+    $ apt-get install cgroup-lite cgroup-tools cgroupfs-mount
+    ```
 
-Additionally not all Ubuntu images are going to have the `memory` cgroup enabled in the Kernel by default, one simple way to enable this is by adding it to the Kernel command line.
+2. Enable the `memory` cgroup in the Kernel by adding it to the Kernel command
+   line.
 
-Open the file `/boot/firmware/cmdline.txt` which is responsible for managing the Kernel parameters, and ensure that the following parameters exist (add them if not):
+3. Open the file `/boot/firmware/cmdline.txt` (responsible for managing
+   the Kernel parameters), and confirm that the following parameters exist (and
+   add them as necessary):
 
-```shell
-cgroup_enable=cpuset cgroup_enable=memory cgroup_memory=1
-```
+    ```shell
+    cgroup_enable=cpuset cgroup_enable=memory cgroup_memory=1
+    ```
 
-Make sure you `reboot` each node to ensure the `memory` cgroup is loaded.
+4. Be sure to `reboot` each node to ensure the `memory` cgroup is loaded.
 
-### Swap (Optional)
+#### Swap (Optional)
 
-While this is _technical optional_ if you have limited RAM (like < 2GB) it can be helpful to enable swap to ease some memory pressure.
+While swap is _technical optional_, enable it to ease memory pressure.
 
-You can create a swapfile by running the following:
+1. To create a swapfile:
 
-```shell
-fallocate -l 2G /swapfile
-chmod 0600 /swapfile
-mkswap /swapfile
-swapon -a
-```
+    ```shell
+    fallocate -l 2G /swapfile
+    chmod 0600 /swapfile
+    mkswap /swapfile
+    swapon -a
```

Comment:
```suggestion
    fallocate -l 2G /swapfile && \
    chmod 0600 /swapfile && \
    mkswap /swapfile && \
    swapon -a
```

this way it can be copypasted in one go.